spec/include/ItemFanSpeed.vspec

```diff
@@ -0,0 +1,13 @@
+# Copyright (c) 2023 Contributors to COVESA
+#
+# This program and the accompanying materials are made available under the
+# terms of the Mozilla Public License 2.0 which is available at
+# https://www.mozilla.org/en-US/MPL/2.0/
+#
+# SPDX-License-Identifier: MPL-2.0
+
+FanSpeed:
+  datatype: string
+  type: actuator
+  allowed: ['AUTO', 'Level1', 'Level2','Level3','Level4','Level5']
```

Comment:
Suggest upper case, like `LEVEL_1`or `LEVEL1`. Also add a newline to the end of the file.